sky/check.py

```diff
@@ -78,14 +81,45 @@ def check_one_cloud(cloud_tuple: Tuple[str, clouds.Cloud]) -> None:
                        f'{enabled_clouds_str}[/green]')
 
 
+def get_enabled_clouds(
```

Comment:
How about renaming it to?
```suggestion
def get_cached_enabled_clouds_or_refresh(
```

Reply:
Good point! Changed.